### PR TITLE
Added 'subtract_', 'exp_', 'remainder_' to paddle frontend Math

### LINKED ISSUE #21929: exp_

### LINKED ISSUE #21934: remainder_

### LINKED ISSUE #21937: subtract_

### PR DESCRIPTION
**Pull request by shashi278**
close #21937 
close #21929 
close #21934

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 1]
```diff
@@ -379,6 +379,41 @@ def test_paddle_subtract(
     )
 
 
+# subtract_
+@handle_frontend_test(
+    fn_tree="paddle.subtract_",
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:384]: the path should be paddle.math.subtract_

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 2]
```diff
@@ -697,6 +732,41 @@ def test_paddle_remainder(
     )
 
 
+# remainder_
+@handle_frontend_test(
+    fn_tree="paddle.remainder_",
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:737]: the path should be paddle.math.remainder_

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 3]
```diff
@@ -988,6 +1058,34 @@ def test_paddle_exp(
     )
 
 
+# exp_
+@handle_frontend_test(
+    fn_tree="paddle.exp_",
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:1063]: the path should be paddle.math.exp_

**Review by umairjavaid — CHANGES_REQUESTED** [review]
The tests are not passing, probably due to incorrect paths in the test functions. Kindly make sure all the tests are passing. 
```
ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:384: in test_paddle_subtract_
    fn_tree="paddle.subtract_",
ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:405: in test_paddle_subtract_
    helpers.test_frontend_function(
ivy_tests/test_ivy/helpers/function_testing.py:825: in test_frontend_function
    frontend_ret = frontend_fw.__dict__[fn_name](*args_frontend, **kwargs_frontend)
E   KeyError: 'subtract_'
E   Falsifying example: test_paddle_subtract_(
E       on_device='cpu',
E       frontend='paddle',
E       backend_fw='torch',
E       dtype_and_x=(['float64', 'float64'], [array(-1.), array(-1.)]),
E       fn_tree='ivy.functional.frontends.paddle.subtract_',
E       test_flags=FrontendFunctionTestFlags(
E           num_positional_args=0,
E           with_out=False,
E           inplace=False,
E           as_variable=[False],
E           native_arrays=[False],
E           generate_frontend_arrays=False,
E       ),
E   )
E   
E   You can reproduce this example by temporarily adding @reproduce_failure('6.82.4', b'AXicY2AAAkYGCGBEYwMBAABkAAU=') as a decorator on your test case
```

**Comment by shashi278** [comment]
Hi @umairjavaid Thanks for the review. I fixed the tests but now I don't see any detailed test logs for these functions. Are they skipped for some reason? Could you please check?

**Comment by aparajith21** [comment]
Hi! Thanks for contributing. Mostly seems okay and LGTM, but will review in detail once you fix conflicts, please let me know when it is ready for review.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**